<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
   </property>
   <property name="title">
    <string>Cr√©er un nouveau compte :</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>veuiller remplir tous les champs :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>cin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nom">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prenom">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_role">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>role</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_mdp">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_id">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <zorder>pushButton_4</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>le_nom</zorder>
   <zorder>le_prenom</zorder>
   <zorder>le_cin</zorder>
   <zorder>le_email</zorder>
   <zorder>le_role</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>le_mdp</zorder>
   <zorder>label_38</zorder>
   <zorder>le_id</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_5</zorder>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-180</y>
     <width>1251</width>
     <height>911</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/back.png</pixmap>
   </property>
  </widget>
  <widget class="QTableView" name="tab_users">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>461</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:#d9c1bb;


font: 81 8pt &quot;Rockwell Extra Bold&quot;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sup">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>280</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rech">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tri">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:#d9c1bb;


font: 81 8pt &quot;Rockwell Extra Bold&quot;</string>
   </property>
   <item>
    <property name="text">
     <string>default</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>id ‚Üë</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>id ‚Üì</string>
    </property>
   </item>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:#d9c1bb;


font: 81 8pt &quot;Rockwell Extra Bold&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stats">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
   </property>
   <property name="text">
    <string>stats</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imp">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    font: bold 14px;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
   </property>
   <property name="text">
    <string>imprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
   </property>
   <property name="text">
    <string>rechercher</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;</string>
   </property>
   <property name="text">
    <string>trier</string>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>groupBox</zorder>
  <zorder>tab_users</zorder>
  <zorder>sup</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>rech</zorder>
  <zorder>tri</zorder>
  <zorder>graphicsView</zorder>
  <zorder>stats</zorder>
  <zorder>imp</zorder>
  <zorder>label_5</zorder>
  <zorder>label_11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
